spec/requests/api/docs/v3/schools_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "swagger_helper"
+
+RSpec.describe "Schools endpoint", openapi_spec: "v3/swagger.yaml", type: :request do
+  include_context "with authorization for api doc request"
+
+  let(:school) { FactoryBot.create(:school, :eligible) }
+  let(:school_partnership) { FactoryBot.create(:school_partnership, school:) }
+  let(:"filter[cohort]") { school_partnership.contract_period.year }
+
+  before do |example|
+    example.metadata[:example_group][:operation][:parameters] += [{
```

Comment:
Sorry I'm maybe just getting myself confused here 🙂 but why do we need this as well as the `filter_schema_ref`?

Reply:
Hi @ethax-ross both schools endpoints requires a filter[:cohort] to be sent in the query params.. so we need to include filter schema ref for in both endpoints for the swagger yaml to be correct. We're manually adding this param here as well outside the yaml so the specs can pass.